<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectWizardPage</class>
 <widget class="QWizardPage" name="ProjectWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Project definition</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Project type:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="comboBoxProjectType">
     <item>
      <property name="text">
       <string>Infrastructure</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Web application</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wireless</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mobile</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IoT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SCADA</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Location:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditLocation"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonDirectoryChooser">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>comboBoxProjectType</tabstop>
  <tabstop>lineEditLocation</tabstop>
  <tabstop>pushButtonDirectoryChooser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
